website/content/specification/computer-example.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<METASCHEMA xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0">
+<METASCHEMA xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://csrc.nist.gov/ns/oscal/metaschema/1.0 ../../../../metaschema-java/core/metaschema/schema/xml/metaschema.xsd"
```

Comment:
Question: is this the actual xsi:schemaLocation with the `/oscal/` in it? This made me confused.

Reply:
Yes indeed!